From: SeongJae Park <sj@kernel.org>
Date: Mon, 8 Sep 2025 18:19:54 -0700
Subject: [PATCH] mm/damon/tests/core-kunit: fix unintended fall-through in
 quota goal test

The switch case is missing break, and hence causing unanotated
fall-through warning.  Add the break to fix it.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202509082042.24Nb5PCp-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/tests/core-kunit.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mm/damon/tests/core-kunit.h b/mm/damon/tests/core-kunit.h
index 0595f36cc537..f02e57ef362e 100644
--- a/mm/damon/tests/core-kunit.h
+++ b/mm/damon/tests/core-kunit.h
@@ -452,6 +452,7 @@ static void damos_test_commit_quota_goal_ensure_committed(struct kunit *test,
 	case DAMOS_QUOTA_NODE_MEM_USED_BP:
 	case DAMOS_QUOTA_NODE_MEM_FREE_BP:
 		KUNIT_EXPECT_EQ(test, dst->nid, src->nid);
+		break;
 	default:
 		break;
 	}
-- 
2.47.3

